src/dom.js

```diff
@@ -7,7 +7,7 @@ import {
   assertSameDomain,
   type SameDomainWindowType,
   type CrossDomainWindowType,
-} from "@krakenjs/cross-domain-utils/src";
+} from "@krakenjs/cross-domain-utils/dist/esm";
```

Comment:
This is really awesome to see! I really like how `dist/esm` provides the vanilla JS code for other deps to use while still supporting the existing tree shaking/dead code elimination strategy.